app/assets/stylesheets/styles.css.scss

```diff
@@ -0,0 +1,45 @@
+
+body {
+  background: transparent url("texture.png");
+}
+
+#nav_bar {
+  color: black;
+  background-color: white;
+}
+
+#nav_bar a {
```

Comment:
Avoid using id selectors<br>Merge rule `#nav_bar a` with rule on line 6<br>Selector `nav_bar` should be written in lowercase with hyphens